scopes/component/merging/merge-cmd.ts

```diff
@@ -22,18 +22,18 @@ export class MergeCmd implements Command {
   extendedDescription = `merge changes of the remote head into local. when on a lane, merge the remote head of the lane into the local.
 if no ids are specified, all pending-merge components will be merged. (run "bit status" to list them).
 optionally use '--abort' to revert the last merge. to revert a lane merge, use "bit lane merge-abort" command.
-${WILDCARD_HELP('merge')}`;
+${WILDCARD_HELP('merge')}`; // @david does this command auto-snap by default? If so we must document it here
   alias = '';
   options = [
-    ['', 'ours', 'in case of a conflict, override the used version with the current modification'],
-    ['', 'theirs', 'in case of a conflict, override the current modification with the specified version'],
-    ['', 'manual', 'in case of a conflict, leave the files with a conflict state to resolve them manually later'],
-    ['', 'abort', 'in case of an unresolved merge, revert to the state before the merge began'],
+    ['', 'ours', 'in case of a conflict, keep the local modification'],
+    ['', 'theirs', 'in case of a conflict, override the local modification with the specified version'],
+    ['', 'manual', 'in case of a conflict, leave files in conflict state to resolve manually later'],
+    ['', 'abort', 'in case of an unresolved merge, revert to pre-merge state'],
     ['', 'resolve', 'mark an unresolved merge as resolved and create a new snap with the changes'],
-    ['', 'no-snap', 'do not auto snap in case the merge completed without conflicts'],
+    ['', 'no-snap', 'do not auto snap even if the merge completed without conflicts'], // @david please review this change
     ['', 'build', 'in case of snap during the merge, run the build-pipeline (similar to bit snap --build)'],
-    ['', 'verbose', 'show details of components that were not merged legitimately'],
-    ['x', 'skip-dependency-installation', 'do not install packages of the imported components'],
+    ['', 'verbose', 'show details of components that were not merged successfully'],
+    ['x', 'skip-dependency-installation', 'do not install packages of the imported components'], // @david what imported components?
```

Comment:
correct. the `imported components` part was probably copied from somewhere else. this is just about about skip running `bit install`.

Reply:
thanks, removed :)